### ISSUE TITLE
define php.ini /usr/local/etc/php/ and memory limit

### ISSUE THREAD

**Issue by valvin1** [friendica/docker#56]
I add many errors in logs like this one : 
`Cannot allocate memory: AH00159: fork: Unable to fork new process`

it looks like this error is linked to the memory limit in `php.ini` : 

```
memory_limit = 128M
```

I had to copy to : 
```
cp /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
```

and then change `memory_limit` to `256M`

would it be possible to setup a php.ini and dynamically setup memory limit from a variable ?

maybe there are other interesting settings to setup ?

**Comment by nupplaphil** [comment]
This is something I was already planing because of the (already solved) problem with memory leaks during contact removage :-)

**Comment by nupplaphil** [comment]
After investigating a little bit, I came over that we should define a "production-ready" memory_limit, as defined in #67 .

I think in other cases, one should "overwrite" the memory_limit in customizing the docker image based on the needs.

So I'd set this issue to resolved when #67 is working as expected :)
